README.md

```diff
@@ -35,52 +42,79 @@ Windows](https://developer.concordium.software/en/mainnet/net/installation/downl
 
 To build the tool from source, you need the following prerequisites:
 
-* Install the Haskell tool Stack:
-   * Via [GHCup](https://www.haskell.org/ghcup/):
-     * Unix: `curl --proto '=https' --tlsv1.2 -sSf https://get-ghcup.haskell.org | sh`
-     * Windows: [Follow the GHCup install guide](https://www.haskell.org/ghcup/install/#installation).
-   * Or, by itself:
-     * Unix: `curl -sSL https://get.haskellstack.org/ | sh`
-     * Windows: [Follow the Stack install guide](https://docs.haskellstack.org/en/stable/install_and_upgrade/).
-
-* Install Rust version 1.53+:
-   * Unix: `curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh`
-   * Windows: [Follow the Rust install
+- Install the Haskell tool Stack:
+  - Via [GHCup](https://www.haskell.org/ghcup/):
+    - Unix: `curl --proto '=https' --tlsv1.2 -sSf https://get-ghcup.haskell.org | sh`
+    - Windows: [Follow the GHCup install guide](https://www.haskell.org/ghcup/install/#installation).
+  - Or, by itself:
+    - Unix: `curl -sSL https://get.haskellstack.org/ | sh`
+    - Windows: [Follow the Stack install guide](https://docs.haskellstack.org/en/stable/install_and_upgrade/).
+
+- Install Rust version 1.53+:
+  - Unix: `curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh`
+  - Windows: [Follow the Rust install
```

Comment:
And this. There was nothing wrong with what was there.

Reply:
It's about consistency, something markdown linter complains about.